website/docs/r/aml_file_system.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+subcategory: "AML File System"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_aml_file_system"
```

Comment:
Can we rename this resource to be `azurerm_machine_learning_file_system`?  

```suggestion
page_title: "Azure Resource Manager: azurerm_machine_learning_file_system"
```

Reply:
Updated